### DISCUSSION TITLE
Deploy in CLI gets stuck at "Sending deployment creation API request" when connected to VPN

### DISCUSSION THREAD

**Discussion by davesierra** [vercel/community#2762] [Help] [answered]
### Summary

Deployment via CLI gets stuck at "Sending deployment creation API request" when connected to VPN.

### Steps to Reproduce

I run `vercel deploy` and it gets stuck in the progress bar indefinitely.

Then I try `vercel deploy -d` to see the output, and notice it gets stuck at "Sending deployment creation API request"

Try a few times thinking some server hiccup somewhere, no change. I then think "maybe it has something to do with my VPN". Sure enough, I disconnect from my VPN and run it again and it goes through just fine.

I am using ExpressVPN, nothing all too special besides that.

**Comment by swarnava** [comment, marked as answer]
Hi there,

We don't have any widespread issue reported at this moment so sounds like your VPN likely blocking the connection. Please reach out to respective team for more information on this.

**Reply by davesierra** [reply]
Fair enough! I'm ok with just to disconnecting when I deploy, just wanted to have it documented to save someone else some time in case they came across the same issue and searched it. Thanks.
